crates/zk-por-core/src/lib.rs

```diff
@@ -9,3 +9,20 @@ pub mod merkle_sum_tree;
 pub mod parser;
 pub mod recursive_prover;
 pub mod types;
+pub mod util;
+
+use account::Account;
+use once_cell::sync::OnceCell;
+use plonky2::hash::hash_types::HashOut;
+use std::sync::RwLock;
+use types::F;
+
+#[derive(Debug)]
+pub struct GlobalConfig {
+    pub num_of_tokens: usize,
+    pub num_of_batches: usize,
+}
+
+pub static EMPTY_ACCT: OnceCell<Account> = OnceCell::new();
+pub static GLOBAL_CONFIG: OnceCell<GlobalConfig> = OnceCell::new();
+pub static GLOBAL_MST: OnceCell<RwLock<Vec<HashOut<F>>>> = OnceCell::new();
```

Comment:
can provide some utility method implemented GLOBAL_MST, e.g, get_root_hash(batch_id), get_recursive_root_hash(level, offset_at_level)? The main prover needs to retrieve these root hashes to make sure that they are identical to proof public inputs.